services/device-registry-base/src/main/java/org/eclipse/hono/service/management/credentials/DelegatingCredentialsManagementHttpEndpoint.java

```diff
@@ -54,7 +54,10 @@
  */
 public class DelegatingCredentialsManagementHttpEndpoint<S extends CredentialsManagementService> extends AbstractDelegatingRegistryHttpEndpoint<S, ServiceConfigProperties> {
 
-
+    /**
+     * Default file uploads directory used by Vert.x Web.
+     */
+    protected static final String DEFAULT_UPLOADS_DIRECTORY = "/tmp";
```

Comment:
I think it should better be avoided to define such a constant in multiple classes.
The constant could be added to `AbstractHttpEndpoint` and only be used from there.